### PR TITLE
qgis3, PDAL, postgis3: update

### PR DESCRIPTION
**Pull request by nilason**
#### Description

Update `qgis3` to 3.32.2, `qgis3-ltr` to 3.28.10, `PDAL` to 2.5.6 and `postgis3` to 3.4.0.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.6.6 21G646 arm64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: gis/qgis3/Portfile [hunk 1]
```diff
@@ -26,23 +26,23 @@ subport ${name-ltr} {}
 
 if {${subport} eq ${name}} {
     # Latest version
-    github.setup    qgis QGIS 3_32_1 final-
+    github.setup    qgis QGIS 3_32_2 final-
     revision        1
```
Comment (reneeotten) [inline gis/qgis3/Portfile:30]: the revision should be reset to 0 when updating the version

File: gis/qgis3/Portfile [hunk 2]
```diff
@@ -26,23 +26,23 @@ subport ${name-ltr} {}
...
-                    sha256  736bb9b93c72709578c1c1c9fc16456a3b6a75b6decac9e52181a95b3331b146 \
-                    size    187226059
+    checksums       rmd160  ddfec56299bc5bcd2bd574533daeab5db2e5978b \
+                    sha256  000f70092ba78b98deb373e1fd5bebf2af9790c892e6503209b8f9ca0d02b414 \
+                    size    187375374
 } else {
     # LTR version
-    github.setup    qgis QGIS 3_28_9 final-
+    github.setup    qgis QGIS 3_28_10 final-
     revision        1
```
Comment (reneeotten) [inline gis/qgis3/Portfile:39]: same here

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: gis/qgis3/Portfile (hunk 1 above)
Comment (nilason) [reply gis/qgis3/Portfile]: Missed these ones, fixed. Thanks!

**Comment by nilason** [comment]
Added a commit with a new qgis3 default variant for better support of plugins (see https://trac.macports.org/ticket/68016#comment:5).

**Comment by nilason** [comment]
> Added a commit with a new qgis3 default variant for better support of plugins (see https://trac.macports.org/ticket/68016#comment:5).

Should add I didn't include the new dependencies for the new `plugin_support` variant completely out of the blue. I did a conservative selection based on https://github.com/qgis/QGIS-Mac-Packager, https://www.freshports.org/graphics/qgis and https://sources.debian.org/src/qgis/3.28.10%2Bdfsg-1/debian/ .

**Comment by reneeotten** [comment]
thanks @nilason, looks good to me! I guess the only question is whether this should be a default variant, but that's up to the maintainer. What do other package managers do?

**Comment by nilason** [comment]
> thanks @nilason, looks good to me! I guess the only question is whether this should be a default variant, but that's up to the maintainer. What do other package managers do?

The official Mac bundle and FreeBSD go all in without question (installs all plugin dependencies right away), I guess that is what the general user expect (hence the number of new tickets). With a default variant we achieve something similar, but with the possibility to opt out.
The dependencies are all python modules so I guess the size isn't much of an issue.

**Comment by nilason** [comment]
Thank you @reneeotten !

**Pull request merged**